Sprint-1/exercises/decimal.js

```diff
@@ -1,9 +1,15 @@
 const num = 56.5678;
-
+console.log(num);
 // You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
 
 // Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
 // Create a variable called decimalPart and assign to it an expression that evaluates to 0.5678 ( the decimal part of num )
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+const wholeNumberPart = Math.floor(num);
+console.log("wHole number is:"+ wholeNumberPart);
+const decimalPart = num - wholeNumberPart;
+console.log("decimal part is:"+ decimalPart);
```

Comment:
Your solution is correct.
Extra challenge for you: Can you make your program to output the decimal part as `0.5678`? That is, only show 4 decimal places in the output. (Right now, your program is showing `0.5677999999999983`, which is normal due to round-off errors happening in floating point number arithmetic).